internal/packages/assets.go

```diff
@@ -122,7 +123,7 @@ func loadElasticsearchAssets(pkgRootPath string) ([]Asset, error) {
 
 		if dsManifest.Type == dataStreamTypeLogs || dsManifest.Type == dataStreamTypeTraces {
 			elasticsearchDirPath := filepath.Join(filepath.Dir(dsManifestPath), "elasticsearch", "ingest_pipeline")
-			pipelineFiles, _ := os.ReadDir(elasticsearchDirPath)
+			pipelineFiles, _ := common.ListFilesByExtension(elasticsearchDirPath, ".yml", ".json")
```

Comment:
For cases with multiple extensions this could be done like this:
```suggestion
			pipelineFiles, err := filepath.Glob(filepath.Join(elasticsearchDirPath, "?(.yml|.json)")
			...
```

This kind of glob syntax works in bash, I haven't tried it in go.

Reply:
Unfortunately Go doesn't support this syntax, but merging the output of two Globs is trivial.